src/main/java/org/primefaces/component/fileupload/NativeFileUploadDecoder.java

```diff
@@ -97,10 +99,30 @@ private static void decodeAdvanced(FacesContext context, FileUpload fileUpload,
         Part part = request.getPart(clientId);
 
         if (part != null) {
+            String contentRange = request.getHeader("Content-Range");
+
+            Pattern pattern = Pattern.compile("bytes ([0-9]+?)-([0-9]+?)/([0-9]+?)");
```

Comment:
Compile this pattern in a static final CONSTANT for performance so it doesn't have to recompile this pattern on every call.

Reply:
Based on https://tools.ietf.org/html/rfc7233#section-4.2, I would go for `^bytes (\d+)-(\d+)\/(\d+|\*)$`